packages/core/core-flows/src/order/workflows/create-return.ts

```diff
@@ -139,6 +141,37 @@ function validateCustomRefundAmount({
   }
 }
 
+function prepareReceiveItems({
+  receiveNow,
+  returnId,
+  items,
+  createdBy,
+}: {
+  receiveNow: boolean
```

Comment:
What is the meaning of `receiveNow`?

Reply:
If true, the return is created and marked as received in the same flow.

Reply:
And in that case, why don't we set the `created_by` and `items`? Shouldn't we have that information regardless, and manage whether something is received or not by a status?

Reply:
`receive_now` is the property used in v1 API. I just kept the name.
I didn't understand the `created_by` and `items` question. This would be a conditional step, as we don't have it yet, we send an empty `items` so the step is skipped

Reply:
ah ok understood, I thought they are stored in the DB 👍